### PR TITLE
[RDY] Changed direct usage of materials with corresponding requirements

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
# What was changed:

- changed direct usages of glue and thread with `adhesive` and `filament` requirements.

### CODE REVIEW

File: data/json/recipes/recipe_ammo.json [hunk 1]
```diff
@@ -177,12 +177,8 @@
   ],
   "components" : [
     [
-      [ "superglue", 1 ],
-      [ "bone_glue", 1 ],
-      [ "thread", 20 ],
-      [ "yarn", 20 ],
-      [ "sinew", 20 ],
-      [ "plant_fibre", 20 ]
+      [ "adhesive", 1, "LIST" ],
```
Comment (Cyrano7) [inline data/json/recipes/recipe_ammo.json]: Duct tape is part of the adhesive requirement but it's in this recipe already with the fletching part of the recipe. Does that have any effect on components needed to make the arrow? Last time I checked it caused some wonky behavior with how much of that component was used.

File: data/json/recipes/weapon/explosive.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "autolearn": true,
     "qualities" : [ { "id": "CUT", "level": 1 } ],
     "components": [
-      [ [ "thread", 10 ] ],
+      [ [ "filament", 10, "LIST" ] ],
```
Comment (Cyrano7) [inline data/json/recipes/weapon/explosive.json]: I believe this part was suppose to represent the fuse. Does specifically the sinew burn like this?

File: data/json/recipes/recipe_ammo.json (hunk 1 above)
Comment (ZhilkinSerg) [reply data/json/recipes/recipe_ammo.json]: Duct tape will be displayed twice in the components list: 

default

If you have multiple components for one of the LIST - you will be asked which alternative to use:

default

default

Components are spent correctly.

File: data/json/recipes/weapon/explosive.json (hunk 2 above)
Comment (ZhilkinSerg) [reply data/json/recipes/weapon/explosive.json]: Sinews are organic, so they certainly burn, but not as good as threads.

Comment (kevingranade) [reply data/json/recipes/weapon/explosive.json]: Good call, in reality even the string is pushing it, in reality fuses are impregnated with gunpowder or other compounds to increase burning temperature, an ordinary cotton string would be slow and unreliable.

**Pull request merged**